svn/trunk/SlickEngagement_Plugin.php

```diff
@@ -391,21 +293,65 @@ public function removeSemicolons($value)
         return str_replace(';', ' ', $value);
     }
 
+    public function getPageBootData() {
+      $siteCode = trim($this->getOption('SiteCode'));
+      if ($siteCode) {
+        global $wp;
+
+        $page_url = (isset($_SERVER['HTTPS']) ? "https" : "http") . "://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]";
```

Comment:
Does the code the looks at the "page url" care whether or not it's http or https? I think we can just remove that part and only pass in the host + path + query string.

Reply:
removed